lib/web_ui/lib/src/engine/canvaskit/initialization.dart

```diff
@@ -15,11 +15,15 @@ import 'canvaskit_api.dart';
 import 'fonts.dart';
 
 /// Whether to use CanvasKit as the rendering backend.
-bool get useCanvasKit => FlutterConfiguration.flutterWebAutoDetect ? _detectRenderer() : FlutterConfiguration.useSkia;
+final bool useCanvasKit = FlutterConfiguration.flutterWebAutoDetect
+  ? _detectedRenderer
+  : FlutterConfiguration.useSkia;
 
 /// Returns true if CanvasKit is used.
 ///
 /// Otherwise, returns false.
+final bool _detectedRenderer = _detectRenderer();
+
 bool _detectRenderer() {
```

Comment:
while we're here it would be nice to rename this something like `hasCanvasKit`. The currently name doesn't indicate what the boolean represents.

Reply:
done